doc/ruleSets/PER-CS1.0.rst

```diff
@@ -2,10 +2,16 @@
 Rule set ``@PER-CS1.0``
 =======================
 
-**This ruleset is deprecated** in favour of `@PER-CS2.0`.
-
 Rules that follow `PER Coding Style 1.0 <https://www.php-fig.org/per/coding-style/>`_.
 
+Warning
+-------
+
+This rule set is deprecated and will be removed in next major version
```

Comment:
Maybe we should have final dot here

Reply:
This is 1:1 like in the deprecated fixers ([example](https://cs.symfony.com/doc/rules/control_structure/no_trailing_comma_in_list_call.html)), and if I remember correctly headers should not have punctuation.

BTW. I've changed to "IN next major version", because for fixers it's "ON next major version". Which one should I keep?

Reply:
Why headers shouldn't have punctuation ?
Is it rally a header after all? Header here is "warning" imho

Reply:
It _is_ a header, look line below (underline is added [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7288/files#diff-3da6626ba31622844f8654f056d574f19245761233992779c5e270fcbfe1778eR87)). I remembered it because of [`markdownlint` rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/md026.md).